nx.json

```diff
@@ -0,0 +1,29 @@
+{
+    "extends": "nx/presets/npm.json",
+    "tasksRunnerOptions": {
+        "default": {
+            "runner": "nx/tasks-runners/default",
+            "options": {
+                "cacheDirectory": "./.nx-cache",
+                "cacheableOperations": ["build", "dev"]
+            }
+        }
+    },
+    "targetDefaults": {
+        "build": {
+            "dependsOn": ["^build"]
+        },
+        "dev": {
+            "dependsOn": ["^build"]
+        },
+        "test": {
+            "dependsOn": ["build"]
+        },
+        "sauce": {
+            "dependsOn": ["^build"]
+        }
```

Comment:
Is it still relevant to have these in here? It looks like `nx` is not running `test` or `sauce` now.

Reply:
I meant to leave a comment about them, these two targets are actually called by nx [here](https://github.com/salesforce/lwc/blob/3ed886ddaeaa28eb64d48597565ff6ecf6c82542/package.json#L22-L23).

For `test`, I left it in so nx would build the dependencies and then build `@lwc/perf-benchmarks` before running the tests.

For `sauce` it builds the dependencies before running the sauce target.

Reply:
Could you add a comment about this? I know it's JSON, but usually `"//": "comment"` works fine.